zingolib/src/wallet/transactions/recording.rs

```diff
@@ -24,85 +24,18 @@ use crate::{
 
 use super::TxMapAndMaybeTrees;
 impl TxMapAndMaybeTrees {
-    pub fn remove_txids(&mut self, txids_to_remove: Vec<TxId>) {
-        for txid in &txids_to_remove {
-            self.transaction_records_by_id.remove(txid);
-        }
+    pub fn invalidate_txids(&mut self, txids_to_remove: Vec<TxId>) {
```

Comment:
I find this pattern of the external type exposing the exact same interface as the internal wrap smelly...    it sniffs funny.

Reply:
actually, its not exact. the outer type deals with witness trees too.

Reply:
otherwise i would just skip this layer

Reply:
I dont understand this reply. why can't we call `invalidate_txids` directly on `transaction_records_by_id` instead of adding this empty wrapper?

Reply:
oops, there was an empty wrapper there! fixed!